mcpwm_foc.c

```diff
@@ -1008,7 +1009,8 @@ void mcpwm_foc_set_openloop_phase(float current, float phase) {
 						  motor_now()->m_conf->l_current_max * motor_now()->m_conf->l_current_max_scale);
 
 	motor_now()->m_control_mode = CONTROL_MODE_OPENLOOP_PHASE;
-	motor_now()->m_iq_set = current;
+	motor_now()->m_id_set = current;
```

Comment:
What changed here between these two lines? Looks like nothing, so maybe just a line ending?

I would use `git gui` to remove that add/subtraction so that the changeset is a little smaller/cleaner.